pkg/config/config.go

```diff
@@ -150,8 +152,16 @@ func (operatorConfig *NSXOperatorConfig) GetTokenProvider() auth.TokenProvider {
 }
 
 func (operatorConfig *NSXOperatorConfig) createTokenProvider() error {
+	log.Info("try to load VC host CA")
+	vcCaCert, err := ioutil.ReadFile(vcHostCACertPath)
+	// If operatorConfig.VCInsecure is false, tls will the CA to verify the server
+	// certificate. If loading CA failed, tls will use the CA on local host
+	if err != nil {
+		log.Info("fail to load CA cert from file", "error", err)
+	}
```

Comment:
Do we load VC cert only when insecure = false?

Reply:
yes. 
updated.